src/components/Background/Background.scss

```diff
@@ -0,0 +1,40 @@
+@use "~stylesheets/typography";
+
+.Background {
+  position: relative;
+  background: var(--darkmode);
+  height: 3000px;
+  width: 100%;
+  z-index: -3;
+
+  &__wire {
+    z-index: -1;
+    position: absolute;
+    width: 100%;
+    bottom: calc(100% / 7 * 3);
+
+    &--variant {
+      position: absolute;
+      width: 100%;
+      bottom: 0;
+    }
+  }
+
+  &__children {
+    z-index: 999;
```

Comment:
Can we also change to to something more realistic? Like 0 or 1? My components have z-indices too, but I'm not sure how that might clash.